cli/src/pcluster/schemas/common_schema.py

```diff
@@ -46,6 +46,14 @@ def validate_json_format(data):
     return True
 
 
+def validate_no_reserved_tag(tags):
+    """Validate there is no tag with reserved prefix."""
+    if tags:
+        for tag in tags:
+            if tag.key.startswith("parallelcluster:"):
```

Comment:
let's create a constant for TAG_PREFIX to avoid typos and re-use it for other tags.